simulation_ws/src/sagemaker_rl_agent/markov/environments/object_tracker_env.py

```diff
@@ -63,14 +63,16 @@ def __init__(self):
                                             shape=(screen_height, screen_width, 3), dtype=np.uint8)
 
         #ROS initialization
-        self.ack_publisher = rospy.Publisher('/cmd_vel', Twist, queue_size=100)
-        self.gazebo_model_state_service = rospy.ServiceProxy('/gazebo/set_model_state', SetModelState)
-        rospy.init_node('rl_coach', anonymous=True)
+        rclpy.init()
+        self.node = rclpy.create_node('rl_coach')
+
+        self.ack_publisher = self.node.create_publisher(Twist, '/cmd_vel', 10)
+        self.gazebo_model_state_service = self.node.create_client(SetEntityState, '/set_entity_state')
 
         #Subscribe to ROS topics and register callbacks
-        rospy.Subscriber('/odom', Odometry, self.callback_position)
-        rospy.Subscriber('/camera/rgb/image_raw', sensor_image, self.callback_image)
-        self.aws_region = rospy.get_param('ROS_AWS_REGION')
+        self.node.create_subscription(Odometry, '/odom', self.callback_position, 10)
+        self.node.create_subscription(sensor_image, '/camera/image_raw', self.callback_image, 10)
```

Comment:
Any reason for renaming `/camera/rgb/image_raw` to `/camera/image_raw`?

Reply:
https://github.com/Rohita83/aws-robomaker-sample-application-objecttracker/blob/aa78176fd6b08b7fcbe1cf3548ed4ae45460e8c1/simulation_ws/src/turtlebot3_description_reduced_mesh/models/turtlebot3_waffle_pi/model.sdf#L401

<plugin name="camera_driver" filename="libgazebo_ros_camera.so">

In waffle_pi Model .sdf file, We have used camera plugin (libgazebo_ros_camera.so) with default image topic name (/camera/image_raw) instead of customize topic name (/camera/rgb/image_raw) as was earliar. 

If you suggest then We can customize topic name for the same.